src/workloads/contrib/qe_range_testing/workloads/local/experiment2_c0_s1.yml

```diff
@@ -0,0 +1,266 @@
+SchemaVersion: 2018-07-01
```

Comment:
Did you mean to add all these workload files at the same time? It might be good to review them separately.

Reply:
nope, that was a mistake lol, deleted now